jobsdb/jobsdb.go

```diff
@@ -3024,7 +3046,7 @@ func (jd *HandleT) getBackupDSRange() dataSetRangeT {
 		}
 	}
 	if len(dnumList) == 0 {
-		return backupDSRange
+		return &backupDSRange
```

Comment:
What was the need to change the return type to a pointer when we are not modifying this variable?

Reply:
got deep source error, that passing large variable as value & should be passed as pointer.

Reply:
Not sure if this particular struct is a large variable but probably @lvrach and @atzoum can help out here

```
type dataSetRangeT struct {
	minJobID  int64
	maxJobID  int64
	startTime int64
	endTime   int64
	ds        dataSetT
}
```

and 

```
type dataSetT struct {
	JobTable       string `json:"job"`
	JobStatusTable string `json:"status"`
	Index          string `json:"index"`
}
```

Reply:
I don't have a strong opinion on this matter. I understand both the performance benefit of passing references instead of copying values around & the benefit of immutable structures. But anyway, a structure is shallow-copied when passed by value, correct? So immutability can only be achieved under conditions.

In the specific scenario, I tend to agree with @cisse21, but I am OK with keeping the linter happy too!